ui/src/app/core/components/contact-us/contact-us.component.html

```diff
@@ -0,0 +1,69 @@
+<div  fxLayoutAlign="center center" fxLayout="column">
+  <h1>Contact Us</h1>
+  <div class="contact" style="background-image: url(assets/contact-background.png);">
```

Comment:
do not use inline style

Reply:
yes, but background image in sass need all path like"../../../../assets/contact-background.png"
when make it like "assets/contact-background.png" can't see it (make error).